src/main/java/com/invoiced/entity/AbstractEntity.java

```diff
@@ -16,23 +14,37 @@
 
 	private Connection conn;
 	protected Class<T> tClass;
+	protected boolean entityCreated;
```

Comment:
Shouldn't this still be private?

Reply:
Yes, must have changed this when trying something and not changed it back. It will be changed back